### PR TITLE
docs(samples): fixed remoteShare sync between 2 users

### PR DESCRIPTION
**Pull request by emanuallan**
fixes https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-254773, relates to https://github.com/webex/webex-js-sdk/pull/1879/files

### CODE REVIEW

File: packages/node_modules/samples/browser-call-with-screenshare/app.js [hunk 1]
```diff
@@ -305,7 +304,6 @@ function bindMeetingEvents(meeting) {
 
   meeting.on('meeting:startedSharingRemote', () => {
     // Set the source of the video element to the previously stored stream
```
Comment (arun3528) [inline packages/node_modules/samples/browser-call-with-screenshare/app.js:306]: add a check for remoteShareStream then set it else dont set it , Sometimes a user is already sharing initially so tthe vent will come before the actual remote sttreams

File: packages/node_modules/samples/browser-call-with-screenshare/app.js [hunk 2]
```diff
@@ -330,8 +328,7 @@ function bindMeetingEvents(meeting) {
       document.getElementById('remote-view-audio').srcObject = media.stream;
     }
     if (media.type === 'remoteShare') {
-      // Remote share streams become active immediately on join, even if nothing is being shared
-      remoteShareStream = media.stream;
+      document.getElementById('remote-screen').srcObject = media.stream;
```
Comment (arun3528) [inline packages/node_modules/samples/browser-call-with-screenshare/app.js:331]: condition to set the streams , check meeting.shareShare === 'remotet_share"

Comment (emanuallan) [reply packages/node_modules/samples/browser-call-with-screenshare/app.js:331]: isnt this already occuring in line 332: `if (media.type === 'remoteShare')` ?

Comment (emanuallan) [reply packages/node_modules/samples/browser-call-with-screenshare/app.js:331]: I also noticed that while stepping, `meeting:startedSharingRemote` is called before the remoteShareStream is set, so remoteShareStream is always undefined.

**Pull request closed without merging**